loafer/aws/consumer.py

```diff
@@ -15,15 +15,17 @@
 
 class Consumer(object):
 
-    def __init__(self, source, options=None, loop=None):
+    def __init__(self, source, endpoint_url=None, use_ssl=True, options=None, loop=None):
         self.source = source
```

Comment:
Entendi, é porque eu estava pensando em ativar esse endpoint_url/use_ssl usando variável de ambiente, se for uma subclasse vai ficar mais difícil